content/_help/verify-your-identity/verify-your-identity-in-person._en.md

```diff
@@ -16,7 +16,7 @@ Please have the following information ready to enter on Login.gov:
 1. Your State-Issued ID. [Don’t have a state issued ID?](/help/verify-your-identity/accepted-state-issued-identification/)
 2. Social Security Number
 3. Your phone number
-   a. If you cannot complete this step, we can send you a one-time code by mail which takes approximately 3-5 days.
+   a. If you cannot complete this step, we can send you a verification code by mail which takes approximately 3-5 days.
```

Comment:
This either needs to be 3-7 business days or 5 to 10 days.

Reply:
Also to be addressed in the 5 to 10 days update.